src/main/java/frc/robot/commands/AimWithLimelight.java

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drive;
+import frc.robot.subsystems.Limelight;
+
+public class AimWithLimelight extends Command {
+  private Drive drive;
+  private Limelight limelight;
+
+  private boolean finished;
+  private int counter;
+
+  private double steerStrength,
+      distanceFromTarget,
+      mountHeight,
+      mountAngle,
+      driveStrength,
+      speedLimit,
+      turnDoneThreshold,
+      distanceDoneThreshold,
+      targetHeight;
+
+  /**
+   * Creates a new command to aim with the Limelight.
+   *
+   * @param limelight The Limelight subsystem to aim with.
+   * @param steerStrength How hard to turn towards the target; between 0 and 1.
+   * @param distanceFromTarget How far from in inches we want to be from the target.
+   * @param mountHeight The height of the center of the Limelight lens off the floor.
+   * @param mountAngle The number of degrees the Limelight is mounted back from perfectly vertical.
+   *     Positive means rotated such that the lens is facing up, and not down.
+   * @param driveStrength How hard to drive towards the target; between 0 and 1.
+   * @param speedLimit Basic speed limit to make sure we don't drive too fast. Percentage of max
```

Comment:
Clarify this is also between 0 and 1 so people don't pass it 70 and get confused why it's not working